VMBackup/main/handle.py

```diff
@@ -668,6 +668,10 @@ def enable():
     try:
         hutil.do_parse_context('Enable', configSeqNo)
         backup_logger.log('starting enable', True)
+        try:
+            create_host_based_service()
```

Comment:
can we add a check that long running extension setting should be enabled before we invoke the host based service. Otherwise we can skip the steps.

Reply:
This would be a CRP passed setting I am assuming?